gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/types.ts

```diff
@@ -0,0 +1,81 @@
+export type OffchainConfig = {
+  deltaProgressNanoseconds: number
+  deltaResendNanoseconds: number
+  deltaRoundNanoseconds: number
+  deltaGraceNanoseconds: number
+  deltaStageNanoseconds: number
+  rMax: number
+  s: number[]
+  offchainPublicKeys: string[]
+  peerIds: string[]
+  reportingPluginConfig: {
+    alphaReportInfinite: boolean
+    alphaReportPpb: number
+    alphaAcceptInfinite: boolean
+    alphaAcceptPpb: number
+    deltaCNanoseconds: number
+  }
+  maxDurationQueryNanoseconds: number
+  maxDurationObservationNanoseconds: number
+  maxDurationReportNanoseconds: number
+  maxDurationShouldAcceptFinalizedReportNanoseconds: number
+  maxDurationShouldTransmitAcceptedReportNanoseconds: number
+  configPublicKeys: string[]
+}
+
+export type InspectionInput = {
+  description: string
+  decimals: string | number
+  minAnswer: string | number
+  maxAnswer: string | number
+  transmitters: string[]
+  payees: string[]
+  signers: string[]
+  offchainConfig: OffchainConfig
+  billingAccessController: string
+  requesterAccessController: string
+  billing: {
+    observationPaymentGjuels: string
+    transmissionPaymentGjuels: string
+  }
+}
+
+export const emptyInspectionInput: InspectionInput = {
```

Comment:
I don't think this is necessary. The only use case seems here: https://github.com/smartcontractkit/chainlink-solana/pull/296/files#diff-f922fe4f599b5db4b4ac741811f33d24961406183e46286ce9e564a4144da114R30
But why don't we simply give `makeInput` the possibility to return `undefined`?
```ts
makeInput = (userInput): Input | undefined => {...}

...
   const input = this.makeInput(this.flags.input)
   if (!input) { // something
```